menu_konsolowe.py

```diff
@@ -76,10 +43,59 @@ def admin_board():
         elif user_choice == "6":
             print("6. EDYCJA DANYCH KONTAKTOWYCH")
         elif user_choice == "7":
-            print("7. WYJŚCIE Z APLIKACJI/WYLOGOWANIE")
+            print("7. ZAPISZ ZMIANY")
+            # save_tasks_to_file()
+        elif user_choice != "8":
+            print(f"Przepraszam, wybrałeś {user_choice}, nie jest to poprawny wybór")
+
+
+def print_menu_customer():
+    print("1. NASZA OFERTA")
+    print("2. SZCZEGÓŁOWE SPECYFIKACJE ORAZ CENNIK USŁUG")
+    print("3. DOSTĘPNOŚĆ BIUREK/STANOWISK")
+    print("4. REZERWACJA BIURKA")
+    print("5. ANULOWANIE REZERWACJI")
+    print("6. DANE KONTAKTOWE BIURA")
+    print("7. REGULAMIN USŁUGI I OPCJE PŁATNOŚCI")
+    print("8. WYJŚCIE Z APLIKACJI")
+
+
+def print_menu_admin():
+    print("1. LISTA REZERWACJI I DANE SUMARYCZNE")
+    print("2. DODWANIE BIURKA/STANOWISKA")
+    print("3. USUWANIE BIURKA/STANOWISKA")
+    print("4. ANULOWANIE REZERWACJI")
+    print("5. EDYCJA REGULAMINU USŁUG")
+    print("6. EDYCJA DANYCH KONTAKTOWYCH")
+    print("7. WYJŚCIE Z APLIKACJI/WYLOGOWANIE")
+    print("8. WYJŚCIE Z APLIKACJI")
+
+
+def main_menu():
+    user_choice = ""
+
+    while user_choice != "3":
+        if user_choice == "1":
+            print("Panel Klienta")
+            print("Zalogowałeś się do panelu klienta")
+
+            customer_board()
+
+        elif user_choice == "2":
+            print("PANEL ADMINISTRATORA ")
+            print("Zalogowałeś się do panelu administratora")
+
+            admin_board()
+
         else:
             print(f"Przepraszam, wybrałeś {user_choice}, nie jest to poprawny wybór")
 
+        print("1. Zaloguj się jako klienta")
```

Comment:
to powinno byc na gorze pętli tak jak w innych waszych menu :) Po za tym wszystko okej